event_routing_backends/processors/xapi/event_transformers/enrollment_events.py

```diff
@@ -14,9 +14,24 @@ class BaseEnrollmentTransformer(XApiTransformer):
     """
     Base transformer for enrollment events.
     """
-    additional_fields = ()
+    additional_fields = ('context', )
     event_version = 1.0
 
+    def get_context(self):
```

Comment:
We don't need this method now since it is defined in base transformer

Reply:
@ziafazal base transformer function returning contextActivities as well. But in case of enrollments transformer we don't need contextActivities